presto-parser/src/main/java/com/facebook/presto/sql/tree/Node.java

```diff
@@ -13,13 +13,30 @@
  */
 package com.facebook.presto.sql.tree;
 
+import com.google.common.base.Equivalence;
+
 import java.util.List;
 import java.util.Optional;
 
 import static java.util.Objects.requireNonNull;
 
 public abstract class Node
 {
+    public static final Equivalence<Node> NODE_VALUE_EQUIVALENCE = new Equivalence<Node>()
```

Comment:
How come `NODE_VALUE_EQUIVALENCE` is unused? I would image `com.facebook.presto.sql.planner.TranslationMap` would be one place that requires it